fabulaws/library/wsgiautoscale/templates/supervisor.conf

```diff
@@ -8,7 +8,7 @@ autostart=true
 autorestart=true
 stdout_logfile={{ log_dir }}/gunicorn.log
 redirect_stderr=true
-environment=NEW_RELIC_CONFIG_FILE='{{ services }}/newrelic-{{ environment }}-web.ini'
+environment=NEW_RELIC_CONFIG_FILE='{{ services }}/newrelic-{{ environment }}-web.ini',SECRET_KEY='{{ secret_key }}'
```

Comment:
This seems more modern, but given all other secrets are saved directly to local_settings.py, perhaps stick w/that paradigm for now? Alternatively, update local_settings.py to use this env variable.